docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/index.md

```diff
@@ -350,6 +350,102 @@ dbt run --selector snowplow_web
+
+```yml
+# dbt_project.yml
+...
+vars:
+  snowplow_ecommerce:
+    snowplow__atomic_schema: schema_with_snowplow_events
+    snowplow__database: database_with_snowplow_events
+    snowplow__events_table: table_of_snowplow_events
+```
 
+:::info Databricks only
 
+Please note that your `target.database` is NULL if using Databricks. In Databricks, schemas and databases are used interchangeably and in the dbt implementation of Databricks therefore we always use the schema value, so adjust your `snowplow__atomic_schema` value if you need to.
+
+:::
+
+#### 3. Enable desired modules
+
+The ecommerce package has the option to join in data from the following 1 Snowplow module:
+
+- App Errors module -- Details relating to app errors that occur during sessions
+
+By default this module is **disabled** in the ecommerce package. Assuming you have the enrichments turned on in your Snowplow pipeline, to enable the module within the package please modify the following in your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+...
+vars:
+  snowplow_ecommerce:
+    snowplow__enable_app_errors_module: true
+```
+
+#### 4. Filter your data set
+
+You can specify both `start_date` at which to start processing events and the `app_id`'s to filter for. By default the `start_date` is set to `2020-01-01` and all `app_id`'s are selected. To change this please add/modify the following in your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+...
+vars:
+  snowplow_ecommerce:
+    snowplow__start_date: 'yyyy-mm-dd'
+    snowplow__app_id: ['my_app_1','my_app_2']
+```
+#### 5. Additional vendor specific configuration
+
+:::info BigQuery Only
+Verify which column your events table is partitioned on. It will likely be partitioned on `collector_tstamp` or `derived_tstamp`. If it is partitioned on `collector_tstamp` you should set `snowplow__derived_tstamp_partitioned` to `false`. This will ensure only the `collector_tstamp` column is used for partition pruning when querying the events table:
+
+```yml
+# dbt_project.yml
+...
+vars:
+  snowplow_ecommerce:
+    snowplow__derived_tstamp_partitioned: false
+```
+
+:::
+
+:::info Databricks only - setting the databricks_catalog
+
+Add the following variable to your dbt project's `dbt_project.yml` file
+
+```yml
+# dbt_project.yml
+...
+vars:
+  snowplow_ecommerce:
+    snowplow__databricks_catalog: 'hive_metastore'
+```
+Depending on the use case it should either be the catalog (for Unity Catalog users from databricks connector 1.1.1 onwards, defaulted to 'hive_metastore') or the same value as your `snowplow__atomic_schema` (unless changed it should be 'atomic'). This is needed to handle the database property within `models/base/src_base.yml`.
+
+**A more detailed explanation for how to set up your Databricks configuration properly can be found in [Unity Catalog support](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/index.md#unity-catalog-support).**
+
+:::
+
```

Comment:
Assume we're adding databricks support before we release the docs?

Reply:
Probably not, have removed!